src/components/Select/Select.scss

```diff
@@ -103,4 +103,18 @@
     width: 0;
     height: 0;
   }
+
+  &__selected-color {
+    border-color: color(ink);
+    border-style: solid;
+    border-radius: 50%;
+    border-width: calc(space(1) / 4);
```

Comment:
it's fine to use `1px` for borders

Reply:
avoiding `px` as much as I can;

I can, as well, set a variable for this. _(I'm overcomplicating the overcomplicated)_

Reply:
For spacing it makes sense to avoid pixels, but for border it's completely fine. If you search the code for `border: 1px` it's used on a lot of other places as well.